src/holofuel/pages/Inbox/Inbox.js

```diff
@@ -374,13 +373,13 @@ function NewTransactionModal ({ handleClose, isNewTransactionModalVisible }) {
 export function ConfirmationModal ({ transaction, handleClose, declineTransaction, refundTransaction, payTransaction, setCounterpartyNotFound, counterpartyNotFound }) {
   const { newMessage } = useFlashMessageContext()
   const { id, amount, type, action } = transaction
-  const { counterparty = {} } = transaction
-  const { holofuelCounterparty, loading } = useCounterparty(counterparty.id)
+  const { counterparty = {}, notFound } = transaction
```

Comment:
`notFound` shouldn't come from transaction here, but from `holofuelCounterparty` on the line below. We should also have a test that would catch this mistake.

Reply:
Agreed, but also don't think this should block the merge given all that is dependent.  I am in favor of making a Chore in PT to update/upgrade some of our testing for measures like this.